src/tools/namespace_tool.cpp

```diff
@@ -122,8 +123,16 @@ int NameSpaceTool::RunCommand(const std::string &cmd) {
             return 0;
         }
     } else if (cmd == kCreateCmd) {
-        return core_->CreateFile(fileName, FLAGS_fileLength * mds::kGB,
-                             FLAGS_stripeUnit, FLAGS_stripeCount);
+        if (!FLAGS_dirName.empty() && !FLAGS_fileName.empty()) {
+            std::cout << "Parameter error: dirName and fileName "
+                      << "can't be set at the same time!" << std::endl;
+            return -1;
+        }
+        bool normalFile = FLAGS_dirName.empty();
```

Comment:
If the user both sets `fileName` and `dirName`, it is recommended to prompt an error in this case

Reply:
root@5dd029eb85a9:/curvebs# ./curve_ops_tool create -dirName=/dir12 -fileName=/test -userName=test -password=123
Parameter error: dirName and fileName can't be set at the same time!